test/e2e/apps/disruption.go

```diff
@@ -459,8 +459,9 @@ func createPodsOrDie(cs kubernetes.Interface, ns string, n int) {
 			Spec: v1.PodSpec{
 				Containers: []v1.Container{
 					{
-						Name:  "busybox",
-						Image: imageutils.GetE2EImage(imageutils.EchoServer),
+						Name:    "busybox",
+						Image:   imageutils.GetE2EImage(imageutils.BusyBox),
+						Command: []string{"sleep", "100000"},
```

Comment:
If all we need is a running container, perhaps we could use pause?
```suggestion
						Name:    "donothing",
						Image:   imageutils.GetPauseImageName(),
```

Reply:
Good point. Thanks for the suggestion. Let's see. :)

Reply:
Tests look good with Pause.